### PR TITLE
Added and updated Technion faculty

### PR DESCRIPTION
**Pull request by YuvalFilmus**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR PULL REQUEST WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [X] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [X] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [X] Combine multiple updates to a single institution into a **single PR.**

- [X] Only submit one pull request per institution.

- [X] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [X] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [X] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [X] Check to make sure that you have no spaces after commas, or any missing fields.

- [X] Check to make sure the home page is correct.

- [X] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [X] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [X] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by YuvalFilmus** [comment]
The error concerns the homepage of Dana Drachsler-Cohen.
It is not a real error, since the validation was able to access it after all.
It works fine for me, even from my phone.

File: csrankings-b.csv [hunk 1]
```diff
@@ -173,6 +173,7 @@ Bastian Goldlücke,University of Konstanz,https://www.cvia.uni-konstanz.de/perso
 Bastian Leibe,RWTH Aachen University,http://www.vision.rwth-aachen.de/person/1/,ZcULDB0AAAAJ
 Bastian Rieck,University of Fribourg,https://bastian.rieck.me,La7zuKQAAAAJ
 Bastin Tony Roy Savarimuthu,University of Otago,https://www.otago.ac.nz/school-of-computing/our-people/tony-savarimuthu,0__bwK0AAAAJ
+Batya Kenig,Technion,https://sites.google.com/view/batyakenig/home,nPuXokoAAAAJ
```
Comment (emeryberger) [inline csrankings-b.csv:176]: Please justify inclusion (non CS). You may have done this in the past but you need to do this any time this happens. Thanks.

File: csrankings-k.csv [hunk 2]
```diff
@@ -695,6 +695,7 @@ Kin-hong Wong,Chinese University of Hong Kong,http://www.cse.cuhk.edu.hk/~khwong
 Kiona Ogle,Northern Arizona University,https://nau.edu/siccs/faculty/kiona-ogle,iNqaSrgAAAAJ
 Kiran Balagani,New York Tech,https://site.nyit.edu/bio/kbalagan,bU6NdUYAAAAJ
 Kiriakos N. Kutulakos,University of Toronto,http://www.cs.toronto.edu/~kyros,NOSCHOLARPAGE
+Kiril Solovey,Technion,https://kirilsol.github.io,b7sIEHcAAAAJ
```
Comment (emeryberger) [inline csrankings-k.csv:698]: Not in CS.

File: csrankings-n.csv [hunk 3]
```diff
@@ -28,6 +28,7 @@ Nadarajah Mithulananthan,University of Queensland,http://researchers.uq.edu.au/r
 Nadarajen Veerapen,CRIStAL,https://www.cristal.univ-lille.fr/profil/veerapen,N-1KB5sAAAAJ
 Nadav Amit,Technion,https://nadav.amit.zone/,SxHdnHcAAAAJ
 Nadav Cohen,Tel Aviv University,http://www.cohennadav.com,DmzoCRMAAAAJ
+Nadav Dym,Technion,https://nadavdym.github.io/,qOyXmMYAAAAJ
```
Comment (emeryberger) [inline csrankings-n.csv:31]: Not in CS.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-b.csv (hunk 1 above)
Comment (YuvalFilmus) [reply csrankings-b.csv:176]: Batya Kenig is a researcher in database systems. She publishes exclusively in CS conferences: ICDT, PODS, UAI, SIGMOD, AAMAS, AAAI. Her area is quite close to that of Benny Kimelfeld, who is faculty at the CS department.

File: csrankings-k.csv (hunk 2 above)
Comment (YuvalFilmus) [reply csrankings-k.csv:698]: Kiril Solovey is a researcher in robotics, close in area to Oren Salzman, who is faculty at the CS department. The two have 18 publications together according to DBLP. All his degrees are in CS, and he was Oren's academic brother (they had the same PhD advisor, Dan Halperin). Kiril publishes in CS conferences such as SOCS and AAMAS.

File: csrankings-n.csv (hunk 3 above)
Comment (YuvalFilmus) [reply csrankings-n.csv:31]: Nadav Dym is a researcher specializing in the theory behind deep learning. He publishes mainly in CS conferences such as AAAI, ICML, ICLR, ICCV and NeurIPS (for example, he has 4 papers only at ICML 2024, and one more in AAAI 2024).

**Comment by emeryberger** [comment]
None of the above comments speak to the inclusion criteria: they need to be able to solely advise a PhD student in computer science.

**Pull request closed without merging**